SwiftReflector/FunctionDeclarationWrapperFinder.cs

```diff
@@ -0,0 +1,273 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Linq;
+using Dynamo;
+using ObjCRuntime;
+using SwiftReflector.Demangling;
+using SwiftReflector.SwiftXmlReflection;
+using SwiftReflector.TypeMapping;
+
+namespace SwiftReflector {
+	public class FunctionDeclarationWrapperFinder {
+		TypeMapper typeMapper;
+		WrappingResult wrappingResult;
+
+		public FunctionDeclarationWrapperFinder (TypeMapper mapper, WrappingResult wrappingResult)
+		{
+			typeMapper = Exceptions.ThrowOnNull (mapper, nameof (mapper));
```

Comment:
this. vs not here is inconsistent?

Reply:
the names don't match so I didn't need `this` in the first one?
Yes, the existing tests that were passing are still passing with the new code. This is a bottleneck through which just about everything runs.